MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/utils/version/VersionUtils.java

```diff
@@ -63,7 +59,7 @@
 @SuppressWarnings("deprecation")
 public final class VersionUtils {
 
-  private static boolean isPaper = false, isParticleBuilderSupported = false;
+  private static boolean isPaper, isParticleBuilderSupported = false;
```

Comment:
so particlebuilder could be also the false removed, if starting with one so do for all